src/config/logger.js

```diff
@@ -13,7 +13,7 @@ const serializer = data => {
 
   return `db.${data.coll}.${data.method}(${query}, ${options})`
 }
-
+// TODO: create script to delete files after a certain period of time.
```

Comment:
I think this is not a programming process. This is a management/administrator process.